manifest_assets/manifest.json

```
@@ -1,14 +1,14 @@
 {
-    "manifest_version": "0.12.0",
+    "manifest_version": "0.13.0",
+    "name": "DUMMY VOICEVOX ENGINE",
     "uuid": "c7b58856-bd56-4aa1-afb7-b8415f824b06",
+    "version": "999.999.999",
+    "url": "https://github.com/VOICEVOX/voicevox_engine",
     "command": "run",
     "port": 50021,
+    "icon": "icon.png",
```

```suggestion
    "icon": "manifest_assets/icon.png",
```
そのエンジンのルートからのパスにした方が良いと思いました

---

manifest.jsonがある場所からの相対パスが綺麗かなーと思ったので変えてみました。
が、runコマンドはエンジンのルートからなのにこっちはassetディレクトリからなので、変なことになってますね。。

manifest.jsonを`engine_manifest.json`という名前に変更してルートに配置するとかが丸いかも･･･？
（`engine_manifest`という名前にするのは、エンジンのルートがそのままソフトウェアのディレクトリに展開されるので、何のマニフェストかわからなくなるためです）

---

なるほど、そっちのほうがよさそうです

---

38c807d で対応しました！

---

対応しましたー！